### PR TITLE
Update the alternative login, verify email, and error boundary components to use css variables for colors

### PR DESCRIPTION
**Pull request by sandymcfadden**
### Fix

This is a continuation of moving to CSS variables for colors within the app.
This updates the Alternative Login, Email Verification, and Error Boundary components.

### Test

The easiest way to test these is to force the modals to appear by temporarily editing the `app.tsx` file to always show them.
To display the error boundary you need to introduce an error in code. 

- Smoke test in light and dark mode
- Compare between production and this branch
- Enable the Alternative Login Prompt
- Ensure it still looks the same
- Enable the Email Verification Modal
- Ensure it still looks the same
- Trigger the Error Boundary
- Ensure it still looks the same

### Release

- Updated the Alternative Login, Email Verification, and Error Boundary components to use CSS variables for colors

### CODE REVIEW

File: lib/dialog/index.tsx [hunk 1]
```diff
@@ -23,7 +23,6 @@ export class Dialog extends Component {
       children,
       onDone,
     } = this.props;
-
```
Comment (belcherj) [inline lib/dialog/index.tsx]: was this change intentional?

**Review by belcherj — APPROVED** [review]

File: lib/dialog/index.tsx (hunk 1 above)
Comment (sandymcfadden) [reply lib/dialog/index.tsx]: It was not at all, thank you! I've fixed it.
